### PR TITLE
Add test and some config files.

### PR DESCRIPTION
**Pull request by m-mujica**
I just moved this test https://github.com/stealjs/steal/blob/master/test/test.js#L209-L211 from steal; test is running in local; I'll all the Gruntfile to run it from the command line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+[StealJS](http://stealjs.com/) that makes it easy to work with css files.
+
+## Install
+
+If you're using StealJS you don't have have to install this plugin, it's included by default.
+
+If you're using SystemJS install this as another npm dependency:
+
+```js
+npm install system-css --save-dev
```
Comment (matthewp) [inline README.md]: should change these to steal-css

**Comment by matthewp** [comment]
Cool, let's get CI set up as well, i can help you with that.

**Comment by m-mujica** [comment]
@matthewp wohooo, we have builds now! :) https://travis-ci.org/stealjs/steal-css/builds/64957854

**Comment by matthewp** [comment]
sweet, merge it in.

**Pull request merged**